swap/src/bin/swap_cli.rs

```diff
@@ -107,22 +107,30 @@ async fn main() -> Result<()> {
             )
             .await?;
 
-            let swap_id = Uuid::new_v4();
+            // TODO: Also wait for more funds if balance < dust
+            if bitcoin_wallet.balance().await? == Amount::ZERO {
+                debug!(
+                    "Waiting for BTC at address {}",
```

Comment:
💡 Eventually we could actually query the ASB for minimum and maximum amounts - in the end the CLI will only be able to swap if the ASB allows it. We configure min/max in XMR at the moment, but we could change that in the ASB. Just an idea, but I think that would add to user experience in the future, because the user would know the boundaries beforehand.